frontend/src/helpers/dropdownMappers.ts

```diff
@@ -121,6 +121,32 @@ export const processFromEntitiesData = (allOptions: tempOption[]) => {
 export function getViasGroupLabel(currentIndex: number | null) {
   return currentIndex ? `${ViasGroupLabel}-${currentIndex}` : OriginsGroupLabel;
 }
+export function areArraysOfObjectsEqual(a: any[], b: any[]): boolean {
```

Comment:
I might be missing something, but can't we just get the ids of each object in both arrays, make a set of them, and compare if them?

Reply:
@afonsobspinto @ddelpiano I'm not so sure if in the second video (Screen.Recording.2024-01-09.at.2.48.06.PM.mov) from time 14 seconds is correct.
as I understand from the video the initial is that the 2nd via is connected (explicit) to all on the first via and has an explicit link to 1 origin. Then the link to the origin is removed.

|What I then notice is that the links to the first vias become implicit (3rd state of the checkbox). Imo this is wrong, they should still show an explicit link (checked state)